package.json

```diff
@@ -23,13 +23,16 @@
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "start:android": "react-native run-android --root example/",
     "start:ios": "react-native run-ios --project-path example/ios --scheme AsyncStorageExample",
+    "start:macos": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos",
     "build:e2e:ios": "detox build -c ios",
     "build:e2e:android": "detox build -c android",
+    "build:e2e:macos": "scripts/run_macos_e2e.sh \"build\"",
```

Comment:
Nit: You can use single quotes to avoid having to escape double quotes. Same below.

Reply:
Sure.  Single vs double quotes have different behavior in bash (single quote won't interpolate) but in this case it doesn't matter.